src/spinnaker.cpp

```diff
@@ -227,7 +303,7 @@ struct SpinnakerCamera final : private Camera
 
     void set(struct CameraProperties* properties);
     void get(struct CameraProperties* properties);
-    void get_meta(struct CameraPropertyMetadata* meta) const;
+    void get_meta(struct CameraPropertyMetadata* meta);
```

Comment:
why did this one go non-const?

Reply:
Oops. I did that temporarily for reasons, but then those reasons no longer applied and I forgot to update it back. Added a commit to add the `const` back and will push it soon.